docs/general/events.md

```diff
@@ -43,18 +33,12 @@ that identify the class of Events but not the instance of the Event.
 <!-- semconv event -->
 | Attribute  | Type | Description  | Examples  | Requirement Level |
 |---|---|---|---|---|
-| `event.domain` | string | The domain identifies the business context for the events. [1] | `browser` | Required |
-| `event.name` | string | The name identifies the event. | `click`; `exception` | Required |
-
-**[1]:** Events across different domains may have same `event.name`, yet be unrelated events.
-
-`event.domain` has the following list of well-known values. If one of them applies, then the respective value MUST be used, otherwise a custom value MAY be used.
+| `event.name` | string | Uniquely identifies the event. | `browser.mouse.click`; `browser.exception` | Required |
```

Comment:
Can we find a different example than `browser.exception`? I don't envision having different exception events for different runtimes. How about using the `device.app.lifecycle` event name defined here? https://github.com/open-telemetry/semantic-conventions/blob/main/docs/mobile/events.md

(Side note, we'll want to update the device app.lifecycle event and any others like it that specify a event.domain.

Reply:
+1

Reply:
How about `device.app.lifecycle` or `mobile.app.lifecycle` ?

Reply:
I think we need to decide on what our prefixes should look like before I change this. Do we need to use `browser` and `mobile` or `device`? What other prefixes would be acceptable and how do we tell consumers about them?

Reply:
This PR is all about the merging of `event.domain` and `event.name`. The natural way to merge these is to concatenate `event.domain + event.name`. So let's update the [mobile lifecycle events](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/mobile/events.md#lifecycle-instrumentation) to have event.names of `device.app.lifecycle`.

> What other prefixes would be acceptable and how do we tell consumers about them?

The [attribute naming](https://github.com/open-telemetry/opentelemetry-specification/tree/v1.26.0/specification/common/attribute-naming.md) document you've linked to defines the rules for `event.name`.

Reply:
Should we do that in this PR or a separate PR?

Reply:
I think this PR. The [mobile lifecylce events](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/mobile/events.md#lifecycle-instrumentation) is the only place where an `event.domain` / `event.name` is specified, so should be small in scope.

Reply:
Done.

Reply:
Marking this comment as "unresolved" because `browser.exception` is still used as an example. Can we switch to `device.app.lifecycle` instead?

Reply:
Should be good now.